aten/src/ATen/native/FunctionOfAMatrixUtils.cpp

```diff
@@ -24,15 +24,15 @@ Tensor _compute_linear_combination(const Tensor& input, const Tensor& coefficien
     input.options().memory_format(at::MemoryFormat::Contiguous)
   );
 
-  _compute_linear_combination_out(input, coefficients, output);
+  native::_compute_linear_combination_out(output, input, coefficients);
 
   return output;
 }
 
 // Note: the function is implemented using the __restrict__ memory modifier,
 // which means that if `output` actually is aliased by `input`, the result
 // produced is undefined.
-Tensor _compute_linear_combination_out(const Tensor& input, const Tensor& coefficients, Tensor& output) {
+Tensor& _compute_linear_combination_out(Tensor& output, const Tensor& input, const Tensor& coefficients) {
```

Comment:
Yeah, I was curious why other `out` methods had `output` as the very first argument. Looks like I completely missed  the schema. 

Actually, the `out` is not being used in `matrix_exp`, but it is good to have. I will probably add it to the existing code to have one larger contiguous buffer instead of separate two.